### ISSUE TITLE
Volume gets larger even though 2 players run alternatively.

### ISSUE THREAD

**Issue by AlphaCircle** [google/ExoPlayer#6350]
Before filing a bug:
-----------------------
- Search existing issues, including issues that are closed:
  https://github.com/google/ExoPlayer/issues?q=is%3Aissue
- Consult our developer website, which can be found at https://exoplayer.dev/.
  It provides detailed information about supported formats and devices.
- Learn how to create useful log output by using the EventLogger:
  https://exoplayer.dev/listening-to-player-events.html#using-eventlogger
- Rule out issues in your own code. A good way to do this is to try and
  reproduce the issue in the ExoPlayer demo app. Information about the ExoPlayer
  demo app can be found here:
  http://exoplayer.dev/demo-application.html.

When reporting a bug:
-----------------------
Fill out the sections below, leaving the headers but replacing the content. If
you're unable to provide certain information, please explain why in the relevant
section. We may close issues if they do not include sufficient information.

### [REQUIRED] Issue description
Describe the issue in detail, including observed and expected behavior.

Test Version: 2.9.6
Test Device: Galaxy S8, S9, S10, LG V30, G7

When 2 players are instatiated but not run at the same time but alternatively,  We found the 2nd player's volume gets bigger, roughly twice.  

### [REQUIRED] Reproduction steps
Describe how the issue can be reproduced, ideally using the ExoPlayer demo app
or a small sample app that you’re able to share as source code on GitHub.

The reproduction is quite simple. Let's call the 1st player "A", the 2nd one "B".
0) Initialize A and B player and set volume "0f", min.
1) Run "A" player with volume "1.0f" max. Let's call this volume amount as "normal" of the phone
2) Pause "A" player and run "B" player with volume "1.0f" max.
3) We found the voice volume gets louder by twice roughly.
4) Pause "B" and run "A" player again with volume "1.0f", max.
5) We found the volume from phone goes back to "normal" of the phone.

Can you help me out to figure out this weird thing? 

### [REQUIRED] Link to test content
Provide a JSON snippet for the demo app’s media.exolist.json file, or a link to
media that reproduces the issue. If you don't wish to post it publicly, please
submit the issue, then email the link to dev.exoplayer@gmail.com using a subject
in the format "Issue #1234", where "#1234" should be replaced with your issue
number. Provide all the metadata we'd need to play the content like drm license
urls or similar. If the content is accessible only in certain countries or
regions, please say so.

### [REQUIRED] A full bug report captured from the device
Capture a full bug report using "adb bugreport". Output from "adb logcat" or a
log snippet is NOT sufficient. Please attach the captured bug report as a file.
If you don't wish to post it publicly, please submit the issue, then email the
bug report to dev.exoplayer@gmail.com using a subject in the format
"Issue #1234", where "#1234" should be replaced with your issue number.

### [REQUIRED] Version of ExoPlayer being used
Specify the absolute version number. Avoid using terms such as "latest".

### [REQUIRED] Device(s) and version(s) of Android being used
Specify the devices and versions of Android on which the issue can be
reproduced, and how easily it reproduces. If possible, please test on multiple
devices and Android versions.

**Comment by marcbaechinger** [comment]
Can you provide us with a test app which let's us reproduces this problem? Or if that is not possible, can you run through your reproduction steps and then take a bug report right after running through these steps? 

In both cases it would be very helpful if you have the `EventLogger` set with `player.addListener`, so the player is logging to the console what is going on in detail.

**Comment by AlphaCircle** [comment]
Hello Marcbaechinger
Thanks for your quick response. I think the faster way is the latter option you have. Let me add the EventLogger to my program and get the log message and share that up with you. Let me get back to you soon.

**Comment by AlphaCircle** [comment]
Hello Marcbaechinger
Sorry for late response. I've tested this or that case and try to build up a test apk for your easiness to this issue. During the preparation and test, I found one weird thing that the volume gets louder with a certain video clip (call A clip) and not with another (call B clip)

I tested Clip A and got the log file as attached and verified again the volume is strange. However, when I tested Clip B with the same program of mine, I saw nothing strange about volume loudness. 

As I know, the volume control might be different in generating a video clip. So I tested both A and B clip on PC and paid attention to listening to the volume change during the play time. But nothing eye-catching (ear-catching actually) happens. I know this approach of test goes to nowhere because it is not with 2 players alternatively playing but wanted to verify the volume is high or low relatively with same configuration of PC video player (my PC player is public-available potPlayer)

I wonder if you have any idea or experience on this and if this thing happens to be related to internal logic of exoplayer.

[Log.txt](https://github.com/google/ExoPlayer/files/3564146/Log.txt)
